scene/gui/text_edit.cpp

```diff
@@ -1180,8 +1180,16 @@ void TextEdit::_notification(int p_what) {
 								draw_rect(Rect2(char_ofs + char_margin + ofs_x, yofs + ascent + 2, w, 1), in_selection && override_selected_font_color ? cache.font_selected_color : color);
 							}
 						} else if (draw_tabs && str[j] == '\t') {
-							int yofs = (get_row_height() - cache.tab_icon->get_height()) / 2;
-							cache.tab_icon->draw(ci, Point2(char_ofs + char_margin + ofs_x, ofs_y + yofs), in_selection && override_selected_font_color ? cache.font_selected_color : color);
+#ifdef TOOLS_ENABLED
+							float line_width = Math::round(EDSCALE);
+#else
+							float line_width = 1.0;
+#endif
+							draw_line(
+									Point2(char_ofs + char_margin + ofs_x, ofs_y),
+									Point2(char_ofs + char_margin + ofs_x, ofs_y + get_row_height()),
+									cache.indent_guide_color,
+									line_width);
```

Comment:
This will only draw the lines when using tabs for indentation. It won't work with spaces. Check the condition above (`if (draw_tabs && str[j] == '\t')`).

Reply:
This was already the case before this change; I haven't looked at adding indent guides when indenting with spaces.